WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackErrorViewModel.swift

```diff
@@ -68,7 +68,15 @@ struct JetpackErrorViewModel: ULErrorViewModel {
         let viewModel = JetpackSetupWebViewModel(siteURL: siteURL,
                                                  analytics: analytics,
                                                  onCompletion: jetpackSetupCompletionHandler)
-        let connectionController = AuthenticatedWebViewController(viewModel: viewModel, wporgCredentials: siteCredentials)
+        let credentials: Credentials? = {
+            if let siteCredentials {
+                return .wporg(username: siteCredentials.username,
+                              password: siteCredentials.password,
+                              siteAddress: siteCredentials.password)
```

Comment:
Looks like we are passing `password` to `siteAddress`

Reply:
Fixed in 1e58fb2fe80798b913cee8ba27bb795f17ba6e85.